app/views/products/index.html.erb

```diff
@@ -1,39 +1,8 @@
-<p id="notice"><%= notice %></p>
-
-<h1>Listing Products</h1>
-
-<!-- <table>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Category</th>
-      <th>Description</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @products.each do |product| %>
-      <tr>
-        <td><%= product.name %></td>
-        <td><%= product.category %></td>
-        <td><%= product.description %></td>
-        <td><%= link_to 'Show', product %></td>
-        <td><%= link_to 'Edit', edit_product_path(product) %></td>
-        <td><%= link_to 'Destroy', product, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
-
-<br>
-
-<%= link_to 'New Product', new_product_path %>
-  -->
-
-
-
-
+<h2>Life's too short to spend time worrying about the small stuff!</h2>
+<p>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque in dapibus lacus. Pellentesque posuere mi vel ligula ultrices sagittis. Morbi efficitur aliquam massa, id iaculis sem venenatis eu. Quisque sit amet sem in risus pretium malesuada. Etiam convallis tincidunt sapien vitae consequat. Sed eu volutpat libero, at lobortis lacus. Donec gravida accumsan lorem quis congue. Maecenas auctor tellus at nunc accumsan, id elementum tellus faucibus. Maecenas vitae cursus libero. Pellentesque ultricies nisl dolor, ac porta orci suscipit eu. Ut accumsan rhoncus urna fermentum sodales. Vestibulum a fermentum nunc. Phasellus gravida pharetra purus feugiat vestibulum. Duis eget rutrum metus. Duis efficitur eu mauris eget dapibus. Etiam libero urna, vulputate dignissim erat eu, vehicula malesuada magna.
+</p>
```

Comment:
I love [this gem](https://github.com/sevenwire/forgery) for generating forged data.